File: leetcode/LengthOfLastWord.java
public class LengthOfLastWord {
    public int lengthOfLastWord(String s) {
        s = s.trim(); // ~strip() in python
        String[] words = s.split(" "); // List of words

        if (words.length == 1) {
            return words[0].length();
        }
        return words[words.length - 1].length();
    }

    public static void main(String[] args) {
        LengthOfLastWord len = new LengthOfLastWord();
        String str = "Hello, World!";
        System.out.println(len.lengthOfLastWord(str));
    }
}

File: leetcode/LengthOfLastWord.cpp
using namespace std;
#include <iostream>
#include <vector>

class Solution {
public:
    int lengthOfLastWord(string s) {
        int length = 0;
        bool counting = false;

        for (int i = s.length() - 1; i >= 0; i--) {
            if (s[i] != ' ') {
                counting = true;
                length++;
            }
            else if (counting) {
                break;
            }
        }
        return length;
    }
};

int main() {
    Solution s;
    string str = "Hello, World!   ";
    cout << s.lengthOfLastWord(str) << endl;
}